src/sumo/wrapper/_auth_provider.py

```diff
@@ -235,6 +238,21 @@ def get_token(self):
     pass
 
 
+class AuthProviderSumoToken(AuthProvider):
+    def __init__(self, resource_id):
+        protect_token_cache(resource_id, ".sharedkey")
+        token_path = get_token_path(resource_id, ".sharedkey")
+        with open(token_path, "r") as f:
+            self._token = f.readline().strip()
+        return
+
+    def get_token(self):
+        return self._token
+
+    def get_authorization(self):
+        return {"X-SUMO-Token": self._token}
```

Comment:
Should it be all caps?
Does case matter?

Reply:
Case does not matter, so we are free to write it however we want. In this case, I think SUMO should be in all-caps (because it is an abbreviation), while Token should have the first letter in upper-case.

Reply:
Sumo is not an abbreviation! Even though many people desperately wants it to be :)